js_modules/dagit/src/configeditor/codemirror-yaml/mode.tsx

```diff
@@ -400,26 +406,30 @@ function findAutocomplete(
     for (const parent of parents) {
       const parentTypeDef = available.find(({ name }) => parent.key === name);
 
-      if (!parentTypeDef || !parentTypeDef.configType.name) {
+      if (!parentTypeDef) {
         return [];
       }
 
-      let childTypeName = parentTypeDef.configType.name;
-      let childEntiresUnique = true;
+      let childTypeKey = parentTypeDef.configType.key;
```

Comment:
Hmm thought I'd updated this, surprised tsc was happy with a possbible `null` here!